keda/values.yaml

```
@@ -122,6 +122,11 @@ affinity: {}
 ## Optional priorityClassName for KEDA Operator and Metrics Adapter
 priorityClassName: ""
 
+## The default HTTP timeout in milliseconds that KEDA should use
+## when making requests to external services. Removing this defaults
+## to the hardcoded value (3000)
+httpTimeout: 3000
```

Maybe a nit but what if we use `http.timeout` here to align with the other conventions?

```suggestion
http:
  timeout: 3000
```